master/internal/resourcemanagers/group.go

```diff
@@ -12,10 +12,8 @@ type groupActorStopped struct {
 // group manages the common state for a set of tasks that all share the same scheduling restrictions
 // (e.g. max slots or fair share weight).
 type group struct {
-	handler *actor.Ref
-	// job      *sproto.JobSummary
-	maxSlots  *int
-	weight    float64 // QUESTION why is prio a ptr but weight is not.
-	priority  *int
-	qPosition float64
+	handler  *actor.Ref
+	maxSlots *int
+	weight   float64 // QUESTION why is prio a ptr but weight is not.
+	priority *int
```

Comment:
question - I get that moving qPosition makes it easier to work with, but I'm wondering if we should keep it here as it is a "scheduling restriction". We're also forwarding the move job message to the RP anyways, and we would keep the scheduler code centralized around groups.

Reply:
there is a clear 1 to one relationship between job and qPosition but there is no such strict direct relationship between group and job.

Reply:
tasks when created sometimes have their `.Group` attribute pointed to the job actor but other times it'd be to the allocate actor and that group wouldn't deal with qPosition ever.